Application/Sources/UI/Controllers/ShowAccessContainerViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ShowAccessContainerViewController.swift
+//  PlaySRG
+//
+//  Created by Mustapha Tarek BEN LECHHAB on 30.01.2025.
+//  Copyright © 2025 SRG SSR. All rights reserved.
+//
```

Comment:
We have a standard project header (which should be configured via some obscure `plist` file in the project shared files):

```suggestion
//
//  Copyright (c) SRG SSR. All rights reserved.
//
//  License information is available from the LICENSE file.
//
```